packages/core/lib/combo/custom/index.ts

```diff
@@ -165,6 +165,13 @@ export const extractedAssets = async (roms: DecompressedRoms): Promise<{[k: stri
   GORON_SYMBOL_TEXTURE: await extractRaw(roms, 'oot', 'objects/object_link_child', 0x7288, 16 * 32 * 2),
   GAUNTLET1_TEXTURE: await extractRaw(roms, 'oot', 'objects/object_link_boy', 0x9980, 16 * 32 * 2),
   GAUNTLET2_TEXTURE: await extractRaw(roms, 'oot', 'objects/object_link_boy', 0x9d80, 32 * 32 * 2),
+  ELEGY_SHELL_HUMAN_MOUTH_TEXTURE: await extractRaw(roms, 'mm', 'objects/gameplay_keep', 0x1caf8, 32 * 32 * 2),
```

Comment:
Stop adding random things to custom_keep, this was already mentionned last time (for strength) and this time it should be very easy to keep the extra data nicely contained in an object file using the splitter.

Reply:
Nintendo keeps the textures in `gameplay_keep` so why is this not appropriate?

Reply:
It burns memory for both games for no reason, and use memory even when not used.

Reply:
Updated